backend/src/test/java/gov/cdc/usds/simplereport/integration/SubmitTestResultIntegrationTest.java

```diff
@@ -12,32 +12,44 @@
 import gov.cdc.usds.simplereport.db.model.auxiliary.MultiplexResultInput;
 import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
 import gov.cdc.usds.simplereport.test_util.SliceTestConfiguration;
+import gov.cdc.usds.simplereport.utils.DateGenerator;
+import gov.cdc.usds.simplereport.utils.UUIDGenerator;
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
+import java.time.Instant;
+import java.util.Date;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.UUID;
 import org.apache.commons.io.IOUtils;
 import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 import org.mockito.Captor;
 import org.skyscreamer.jsonassert.JSONAssert;
-import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.context.annotation.Import;
+import org.springframework.test.context.bean.override.mockito.MockitoBean;
 import reactor.core.publisher.Mono;
 
 @SliceTestConfiguration.WithSimpleReportStandardUser
 @Import(SubmitTestResultTestConfig.class)
 class SubmitTestResultIntegrationTest extends BaseGraphqlTest {
 
-  @MockBean(name = "mockClient")
+  @MockitoBean(name = "mockClient")
   QueueAsyncClient queueAsyncClient;
 
+  @MockitoBean DateGenerator dateGenerator;
+  @MockitoBean UUIDGenerator uuidGenerator;
+
   @Captor ArgumentCaptor<String> fhirMessageCaptor;
 
   @Test
   void complete_submit_test_result_flow() throws IOException {
+    Date date = Date.from(Instant.parse("2023-05-24T19:33:06.472Z"));
```

Comment:
What prompted this change? This seems to be new logic in a test or at least moving it from another file?